nncf/torch/graph/transformations/commands.py

```diff
@@ -164,6 +164,25 @@ def requires_graph_rebuild(self):
         return True
 
 
+class PTInsertionTemporaryCommand(PTTransformationCommand):
+    """
+    Insertion operation to the models. Given fn is kept in HookGroups.TEMPORARY group of hooks.
```

Comment:
Could you briefly explain difference between `Hook Groups.PERMANENT` and `Hook Groups.TEMPORARY` and why you added two groups instead of one?